src/components/layouts/Header.tsx

```diff
@@ -16,12 +16,12 @@ export default function Header() {
         updateShowNotificationsScreen={updateShowNotificationsScreen}
       />
       <header>
-        <div className="logo-name">Bench-routes</div>
+        <div className='logo-name'>Bench-routes</div>
         {!showNotification ? (
-          <div className="notification-icon">
+          <div className='notification-icon'>
             <img
-              src="assets/icons/notify-icon.svg"
-              alt="notification"
+              src='assets/icons/notify-icon.svg'
```

Comment:
Eslint made single-quotes to double quotes, tslint making it to single quotes XD

Reply:
yeah..we need to unify some rules.